extensions/prototypes/labgraph_monitor/README.md

```diff
@@ -21,28 +21,31 @@ yarn -v
 
 **Set up the application**
 
+(!) The following tutorial utilizes **yarn** (recommended) however **npm** can be utilized as well.
+
 1. Be sure that you are inside **extensions/prototypes/labgraph_monitor** directory
 
 ```
-cd extensions/prototypes/labgraph_monitor
+labgraph> cd extensions/prototypes/labgraph_monitor
+labgraph\extensions\prototypes\labgraph_monitor>
 ```
 
 2. Install dependencies by running **yarn** command
 
 ```
-yarn
+labgraph\extensions\prototypes\labgraph_monitor> yarn
 ```
 
 3. Test the application by running the following command
 
 ```
-yarn test --watchAll=false
+labgraph\extensions\prototypes\labgraph_monitor> yarn test --watchAll=false
 ```
 
 4. Run the application
 
 ```
-yarn start
+labgraph\extensions\prototypes\labgraph_monitor> yarn start
```

Comment:
An empty UI has launched after running this line.
Reference: https://pasteboard.co/K6noFCNDD7A3.png

Reply:
Do you the icon on the right?
<img width="1314" alt="Screen Shot 2022-04-22 at 2 33 13 PM" src="https://user-images.githubusercontent.com/30349896/164774135-28859b06-c191-4c41-83b9-706cf1a82c8a.png">